docs/sources/operations/zone-ingesters.md

```diff
@@ -7,60 +7,59 @@ weight:
-3. Enable zone awareness on the write path by setting `distributor.zone-awareness-enabled` to true for distributors and rulers.
-4. Wait some time to ensure that the new zone aware ingesters have data for the time period they are queried for (`query_ingesters_within`).
-5. Enable zone awareness on the read path by setting `distributor.zone-awareness-enabled` to true for queriers.
-6. Configure distributors and rulers to exclude ingesters in the `zone-default` so those ingesters no longer receive write traffic via `distributor.excluded-zones`.
-7. Use the shutdown endpoint to flush data from the default ingesters, then scale down and remove the associated StatefulSet.
-8. Clean up any config remaining from the migration.
+1. Configure your existing ingesters to be part of a zone, for example `zone-default`, this will allow us to later exclude them from the write path while still allowing for graceful shutdowns.
+1. Prep for the increase in active streams (due to the way streams are split between ingesters) by increasing the number of active streams allowed for your tenants.
+1. Add and scale up your new zone-aware ingester StatefulSets such that each has 1/3rd of the total number of replicas you want to run.
+1. Enable zone awareness on the write path by setting `distributor.zone-awareness-enabled` to `true` for distributors and rulers.
+1. Wait some time to ensure that the new zone-aware ingesters have data for the time period they are queried for (`query_ingesters_within`).
+1. Enable zone awareness on the read path by setting `distributor.zone-awareness-enabled` to true for queriers.
+1. Configure distributors and rulers to exclude ingesters in the `zone-default` so those ingesters no longer receive write traffic using `distributor.excluded-zones`.
+1. Use the shutdown endpoint to flush data from the default ingesters, then scale down and remove the associated StatefulSet.
+1. Clean up any config remaining from the migration.
 
 ### Detailed Migration Steps
 
 The following are steps to live migrate (no downtime) an existing Loki deployment from a single ingester StatefulSet to 3 zone aware ingester StatefulSets.
 
 These instructions assume you are using the zone aware ingester jsonnet deployment code from this repo, see [here](https://github.com/grafana/loki/blob/main/production/ksonnet/loki/multi-zone.libsonnet). **If you are not using jsonnet see the relevant annotations in some steps that describe how to perform that step manually.**
 
-1. Configure the zone for the existing “ingester” StatefulSet as zone-default by setting multi_zone_default_ingester_zone: true, this allows us to later filter out that zone from the write path.
+1. Configure the zone for the existing ingester StatefulSet as `zone-default` by setting `multi_zone_default_ingester_zone: true`, this allows us to later filter out that zone from the write path.
```

Comment:
Line 39, are we talking about configuration parameters?  Should those be formatted as parameters?

Configure `ingester-pdb` with `maxUnavailable=0` and deploy  3x....